galasa-parent/dev.galasa.framework.api.resources/src/test/java/dev/galasa/framework/api/resources/routes/TestResourcesRoute.java

```diff
@@ -1322,4 +1323,61 @@ public void TestHandlePOSTwithApplyMultipleExistingAndNewAndNullPropertiesReturn
         checkPropertyInNamespace(namespace, propertynametwo, valuetwo);
     }
 
+    @Test
+    public void TestGetErrorsAsJsonReturnsJsonString() throws Exception{
+        // Given...
+        List<String> errors = new ArrayList<String>();
+        errors.add("{\"error_code\":5030,\"error_message\":\"GAL5030E: Error occured when trying to delete Property 'property.5'. Report the problem to your Galasa Ecosystem owner.\"}");
+        errors.add("{\"error_code\":5030,\"error_message\":\"GAL5030E: Error occured when trying to delete Property 'property.1'. Report the problem to your Galasa Ecosystem owner.\"}");
+        setServlet("framework");
+        MockResourcesServlet servlet = getServlet();
+        IFramework framework = servlet.getFramework();
+        ResourcesRoute resourcesRoute = new ResourcesRoute(null, framework);
+
+        // When...
+        String json = resourcesRoute.getErrorsAsJson(errors);
+
+        // Then...
+        String expectedJson = "[\n  {\n    \"error_code\": 5030,\n    \"error_message\": \"GAL5030E: Error occured when trying to delete Property 'property.5'. Report the problem to your Galasa Ecosystem owner.\"\n  },"
+            +"\n  {\n    \"error_code\": 5030,\n    \"error_message\": \"GAL5030E: Error occured when trying to delete Property 'property.1'. Report the problem to your Galasa Ecosystem owner.\"\n  }\n]";
```

Comment:
Might be worth making an expected JSON object and converting that into a string to improve the readability of this and make it less brittle - same for the test below